polaris-react/src/components/IndexTable/IndexTable.stories.tsx

```diff
@@ -7,7 +7,7 @@ import type {
 } from '@shopify/polaris';
 import {
   Icon,
-  HorizontalStack,
+  InlineStack,
```

Comment:
With @shopify/polaris@12.0.0 [has been shipped](https://shopify.slack.com/archives/C4Y8N30KD/p1696832186682759) we no longer have `HorizontalStack` component, so I replaced it with `InlineStack`